cluster/gce/gci/configure-helper.sh

```diff
@@ -425,6 +429,56 @@ function ensure-local-ssds() {
   done
 }
 
+function ensure-local-ssds-ephemeral-storage() {
+  # Get scsi devices
+  local devices=(/dev/disk/by-id/google-local-ssd-*)
+  if [ ! -e ${devices[0]} ]; then
+    echo "No local SCSI SSD disks found."
+    devices=()
+  fi
+  # Get nvme devices
+  for ssd in "/dev/nvme*"; do
+    if [ -e "${ssd}" ]; then
+      # This workaround to find if the NVMe device is a disk is required because
+      # the existing Google images does not expose NVMe devices in /dev/disk/by-id
+      if [[ `udevadm info --query=property --name=${ssd} | grep DEVTYPE | sed "s/DEVTYPE=//"` == "disk" ]]; then
+        devices+=("${ssd}")
+      fi
+    else
+      echo "No local NVMe SSD disks found."
+    fi
+  done
+  if [ "${#devices[@]}" -eq 0 ]; then
+    return
```

Comment:
Can we error or log or something here? I don't know how we can surface node startup problems, but if there is one it would be nice if it were possible to see it clearly in logs or whatever.

Reply:
We are already logging in the lines above. If there are no SSDs, I don't think it should be considered an error.

Reply:
ok, I agree those logs should be enough.